src/services/infrastructure/kratos/kratos.service.ts

```
@@ -0,0 +1,336 @@
+    }
+    return identity[0];
+  }
+
+  /**
+   * Deletes an identity by email.
+   *
+   * @param email - The email of the identity to be deleted.
+   * @returns A promise that resolves to void if the identity is deleted successfully.
+   * @throws UserIdentityNotFoundException if the identity with the specified email is not found.
+   */
+  public async deleteIdentityByEmail(email: string): Promise<void | never> {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)